config/locales/hyrax.it.yml

```diff
@@ -785,10 +785,10 @@ it:
         versions: versioni
         versions_title: Visualizza le versioni precedenti
       groups_description:
-        description_html: L'elenco dei gruppi nel menu a discesa contrassegnato "Seleziona un gruppo" è un elenco di gruppi gestiti dall'utente di cui sei membro. È possibile selezionare un gruppo specifico e assegnare un livello di accesso per un file all'interno di %{application_name}, in modo simile all'aggiunta dei livelli di accesso dell'utente.
+        description_html: L'elenco dei gruppi nel menu a discesa contrassegnato 'Seleziona un gruppo' è un elenco di gruppi gestiti dall'utente di cui sei membro. È possibile selezionare un gruppo specifico e assegnare un livello di accesso per un file all'interno di %{application_name}, in modo simile all'aggiunta dei livelli di accesso dell'utente.
```

Comment:
Since this string has single quotes in it, you should leave these as double quotes

Reply:
Double quotes have been added back in all language files.